test/presubmit-config-check.sh

```diff
@@ -16,43 +16,8 @@
 
 source $(dirname $0)/../scripts/library.sh
 
-# Run diff on the two configuration files.
-# Parameters: $1 - the regenerated temp file.
-#             $2 - the existing file.
-function diff_config_files() {
-  diff --ignore-matching-lines="^# Copyright " "$1" "$2"
```

Comment:
You lost this feature

Reply:
Is it because of this line?
```
# Copyright [[.Year]] The Knative Authors
```

If so, I would rather prefer let it fail when this is different, `update-codegen.sh` can fix it right?

Reply:
Right, this is correct.
We added this because you can't approve a PR (like an automated one) from two days ago if today happens to be Jan 2. It's really annoying.

Reply:
For this once in a year thing which can be fixed by `update-codegen.sh`, my personal preference would be just let it fail instead of this hack. WDYT? @chizhg for opinion as well

Reply:
I was wrong, all build tests start failing in the new year, every single year:
https://github.com/knative/test-infra/pull/1606
It needs to be kept.

Reply:
Thinking about this more, given these files are generated by a computer program, they shouldn't have a copyright at all!
The source files can, but their copyright should be fixed by the author and not dynamically. So this is the appropriate fix: #1833

Reply:
Agree